src/auth/index.ts

```diff
@@ -44,14 +48,13 @@ export async function login(
   }
 }
 
-export async function changePassword(
-  username: string,
+async function changePassword(
+  usr: User,
```

Comment:
nit: would prefer `user` to keep it consistent.

Reply:
Changed because the linter was complaining about having a name matching the type. It does not like 'user: User' which is my preference too. 

Maybe if we turn off the 'prettier' mode?

Reply:
Nevermind, the linter is fine with it now.